scripts/copy-types.js

```diff
@@ -1,12 +0,0 @@
-// A single declaration file CAN NOT represent both CommonJS and ESM module, even if their content is identical.
```

Comment:
This file wasn't referenced anywhere. I don't know if we need to have both `.d.ts` and `.d.cts` files